mautic/mautic
# Fix change date modified If last_active date change

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | 
| Deprecations?                          | 
| BC breaks? (use the c.x branch)        | 
| Automated tests included?              | 
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 



#### Description:

If we change just last_active date during some action, also date_modified of contact  is changed. What does not make sense. Last activity date and date_modified date are different columns with different ways to use. 



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Create email with link and send it
3. Copy tracked link from email
4. Open it in incognito window
5. Check contact profile probably in database and see like date_modified was changed. 
6. After PR is not chang, because we want update just last activity of the contact. His profile was not modified, then last_modified date should to be changed
